packages/cfd/src/Components/__tests__/cfd-account-card.spec.js

```diff
@@ -0,0 +1,486 @@
+import React from 'react';
+import { screen, render, fireEvent } from '@testing-library/react';
```

Comment:
Could you consider alphabetic order for imports

Reply:
On it 👍